content/02.introduction.md

```diff
@@ -44,82 +57,70 @@ The current pandemic of COVID-19, caused by the severe acute respiratory syndrom
-[@doi:10/ggnxb3] found that a higher probability of mortality was associated with older age and higher Sequential Organ Failure Assessment scores, as well as high levels of d-dimer.
-Mortality might be associated with other biomarkers measured in blood samples including lactate dehydrogenase and cardiac troponin I, although these analyses may not have been appropriately corrected for multiple testing.
-They also found that survivors continued to shed the virus for a median of 20 days and a maximum of at least 37 days.
-
-#### Molecular Mechanisms of COVID-19
-
-- How can we evaluate the human response to SARS-CoV-2 on a molecular level?
-- What can omics technologies tell us about changes in the body following SARS-CoV-2 infection?
-
-SARS-CoV-2 is a novel human coronavirus, following SARS-CoV and MERS-CoV to become the third coronavirus causing severe pneumonia in humans to emerge in the 21st century [@doi:10.1016/bs.aivir.2018.01.001]. 
-Therefore, most of our current knowledge of the molecular mechanisms through which SARS-CoV-2 infects cells is obtained from studies on previously identified coronaviruses. 
-The entry of a coronavirus into a target cell is mediated by the binding of the viral spike (S) protein to a specific cellular receptor and the subsequent priming/cleavage of the S protein by cellular proteases [@doi:10/cf8chz; @doi:10.1016/j.antiviral.2020.104792]. 
-Angiotensin-Converting Enzyme 2 (ACE2) and Transmembrane Serine Protease 2 (TMPRSS2) have been identified as a prime receptor and a critical protease, respectively, facilitating SARS-CoV/CoV-2 entry into a target cell [@doi:10.1038/nature02145; @doi:10.1126/science.abb2507; @doi:10.1128/JVI.01542-10; @doi:10.1016/j.cell.2020.02.052; @doi:10.1128/JVI.02232-10]. 
+#### Cell Entry and Replication
+
+Coronavirus virions are spherical with diameters ranging from 100 to 160 nanometers (nm).
+The virion is made up of a lipid envelope in which peplomers of 2-3 spike (S) glycoproteins are anchored, creating a distinctive "crown" shape for which the family of viruses was named [@isbn:0781760607; @doi:10.1128/JVI.80.3.1302–1310.2006].
+The replication process is initiated by the viral spike protein: first, the virus binds to a host cell, and then the viral membrane fuses with the endosomal membrane to release the viral genome into the host cytoplasm.
+The spike of SARS-CoV is structured in three segments: the ectodomain, a transmembrane anchor, and an intracellular tail [@doi:10.1128/JVI.02615-14].
+The ectodomain contain two subdomains known as the S1 and S2 subunits, with the S1 (N-terminal) subunit guiding the binding of the virus to a host cell receptor and the S2 (C-terminal) subunit guiding the fusion process [@doi:10.1128/JVI.02615-14].
+<!-- Is it worth going into more detail about siganl peptides and secretory pathways? Not sure whether this is relevant information.-->
```

Comment:
I think if there are treatments or parthenogenesis traits, mentioned in the other sections, that need that background it might be good to add

Reply:
That makes total sense @ajlee21! I might leave the comment in for now just as a reminder to check against the therapeutics section!